build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -113,6 +113,18 @@ steps:
       yarn gulp package-external-extensions
     displayName: Package External extensions
 
+  - script: |
+      set -e
+      DISPLAY=:10 ./scripts/test.sh --build --tfs "Unit Tests" --runGlob '**/sql/**/*.test.js'
```

Comment:
Is there a reason you're not doing --coverage here?

Although doing that is probably going to overwrite the code coverage file, right? Not sure how merging is going to work either since they'll share some of the same files which might confuse anything consuming it. May need to look into an actual tool for that in that case.